lib/interpreters/compute_several_solutions.rb

```diff
@@ -172,7 +172,7 @@ def self.find_best_heuristic(service_vrp)
 
         custom_heuristics << 'supplied_initial_routes' if vrp.routes.any?
 
-        times = []
+        elpased_array = []
```

Comment:
there is something wrong here :thinking: I remember this exact modification from another PR -- which suggested `elapsed_array` that became `elapsed_times` but here is `elpased_array` again with a typo?

Reply:
@braktar do you understand what happened? Am I hallucinating :confused:

Reply:
It comes with this PR https://github.com/Mapotempo/optimizer-api/pull/359 which not merged yet

Reply:
Did you make overlapping modification in two different commits of two different PRs :sweat: no wonder I am confused -- it was a real deja-vu

Reply:
This PR requires #359 so yeah it overlaps

Reply:
I just realized the commit names !

Reply:
https://github.com/Mapotempo/optimizer-api/pull/359 is merged you can rebase this one :t-rex: